gildniy/WayFarer
# #167122110 View a Single Trip

#### What does this PR do?
See the single trip details
#### Description of Task to be completed?
Have these details listed:
- Trip departure place
- Trip destination place
- Trip date
- Trip time
- Bus plate number
- Driver name
- Image of the bus with buttons to select a seat on the current trip
- A button to get back to the list
- And a button to submit the request of the seat booking
#### How should this be manually tested?
By visiting the `/dashboard.html page` the user will be able to open the single trip by clicking on the `open` button in the list of trips. 
Once on the Single trip item, the user should be able to see the information about the currently opened trip, and can select 1 or more seats of the available seats on the trip (bus image), or can deselect selected seats.
#### Any background context you want to provide?
As the user will need to see the placement of the seat and the number of the seat, a numbering has been provided based on the line the side of the bus. A random number of 40 seats will are in 1 trip bus.
#### What are the relevant pivotal tracker stories?
#167122110
#### Screenshots 
![image](https://user-images.githubusercontent.com/3137192/61014640-c6574a80-a388-11e9-8273-afcf91052a12.png)
